content/docs/react-without-jsx.md

```diff
@@ -38,11 +38,11 @@ ReactDOM.render(
 );
 ```
 
-If you're curious to see more examples of how JSX is converted to JavaScript, you can try out [the online Babel compiler](babel://jsx-simple-example).
+यदि आप अधिक उदाहरण देखने के लिए उत्सुक हैं कि JSX को जावास्क्रिप्ट में कैसे परिवर्तित किया जाता है, तो आप [ऑनलाइन बाबेल कंपाइलर](babel://jsx-simple-example) आज़मा सकते हैं।
 
-The component can either be provided as a string, as a subclass of `React.Component`, or a plain function.
+कोंपोनेंट या एक स्ट्रिंग के रूप में या `React.Component` के उपवर्ग (सब-क्लास) के रूप में या फिर एक सादे फ़ंक्शन के रूप में प्रदान किया जा सकता है।
```

Comment:
`कोंपोनेंट` => `कौम्पोनॅन्ट`

refer glossary for commonly translated words https://github.com/reactjs/hi.reactjs.org/issues/2

Reply:
`उपवर्ग (सब-क्लास)` => `सब-क्लास`

Reply:
The word `कौम्पोनॅन्ट` is incorrect even in glossary. The correct word is `कॉम्पोनेन्ट`. Please suggest.

Reply:
Use the one i have suggested. Since we need to keep consistency.
In that way, when we want to change it, it will just find and replace